packages/core/template/portfolio/contents/projects.md

```diff
@@ -0,0 +1,61 @@
+# Projects
+
+<include src="project1.md"/>
+<include src="project2.md"/>
+
+### Other projects
+
+<div class="row row-cols-1 row-cols-md-3 g-3">
+  <div class="col">
+    <div class="card">
+        <h4 class="card-header">Project title</h4>
+        <div class="card-body">
+            This is another project. 
+        </div>
+        <pic src="https://markbind.org/images/logo-lightbackground.png" alt="markbind-logo" width="90%" height="200"></pic>
+        <div class="card-body">
+            <p class="card-text">Some further elaboration on the project.</p>
+            {{javascript}}
+            {{ vue }}
+        </div>
+        <div class="card-body">
+            <a href="#" class="btn btn-primary">View on Github</a>
```

Comment:
Suggest that this link should go to MarkBind github and the one below should go to MarkBind website

Reply:
Great! @jingting1412 the button next to this seems wrong tho? What is "post"

Reply:
I was thinking that "Devpost" seems a bit inaccurate, since we're linking to documentation, but yeah "post" might be a little confusing too 😓. I'll change it back to Devpost